controller/applicationController.js

```diff
@@ -0,0 +1,22 @@
+const responseHandler = require("../utils/responseHandler");
+const { getAppToken } = require("../utils/auth/tokenGenerator");
+
+const getApplicationToken = async (req, res, next) => {
+  const { applicationId } = req.params;
+  const { adminId } = req.body;
```

Comment:
Great work here! Everything thing is in perfectly implemented. Please, modify the `req.body` to `req.token`. The adminId is extracted by the orgAuthMW from the organizationToken, into the `req.token`. The `req.body` is actually empty. Please effect this change. Thank you